vsc_pdf_template_transformer/lib/models/tpl_pdf_border.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/pdf.dart';
+import 'package:vsc_pdf_template_transformer/models/tpl_pdf_document.dart';
+import 'package:vsc_pdf_template_transformer/utils/evaluator.dart';
+
+part 'tpl_pdf_border.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplPdfBorder {
```

Comment:
This requires the root-level PdfDocument, which you don't have. Please remove it. It's not a widget.